packages/compass-e2e-tests/helpers/commands/get-connect-form-connection-string.ts

```diff
@@ -0,0 +1,18 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function getConnectFormConnectionString(
+  browser: CompassBrowser,
+  isFocused = false
+): Promise<string> {
+  const inputElem = await browser.$(Selectors.ConnectionStringInput);
+  await inputElem.waitForDisplayed();
+  if (isFocused) {
+    await browser.waitUntil(async () => {
+      await inputElem.click();
+      return await inputElem.isFocused();
+    });
+  }
+  const connectionString = await inputElem.getValue();
+  return connectionString;
```

Comment:
just a small stylistic thing, feel free to ignore:

```suggestion
  return await inputElem.getValue();
```